examples/weather-demo/build.rs

```diff
@@ -0,0 +1,7 @@
+use std::env;
+
+fn main() {
+    env::set_var("SLINT_ENABLE_EXPERIMENTAL_FEATURES", "true");
```

Comment:
I'm hesitant about the use of change callbacks in the demos/examples. @ogoffart what's your opinion?

Reply:
Initially, we tried to avoid them. I believe there are at least one or two use cases where I could not find a workaround for not using them and still providing the functionality.
Maybe I'm missing something. Please let me know if you want me to examine this again and provide you with concrete examples.

Reply:
Alright, let's keep the use of the change handlers as they are :-)